cmd/cli/uninstall_mesh.go

```diff
@@ -331,9 +331,6 @@ func (d *uninstallMeshCmd) uninstallValidatingWebhookConfigurations() error {
 func (d *uninstallMeshCmd) uninstallSecrets() error {
 	secrets := []string{
 		d.caBundleSecretName,
-		constants.CrdConverterCertificateSecretName,
```

Comment:
Just for my own understanding, was this ever doing anything not already handled by the cleanup hook?
https://github.com/openservicemesh/osm/blob/f10a3cc7ebb08ab94bf7092fb356d1182740df90/charts/osm/templates/cleanup-hook.yaml#L86